enabler/src/de/schildbach/pte/NetworkId.java

```diff
@@ -83,4 +83,7 @@
 
     // Africa
     GHANA,
+
+    // Nicaragua
+    ESTELI, MANAGUA,
```

Comment:
remove the comma :)

Reply:
Which comma?

Reply:
The one after MANAGUA

Reply:
It should be there, i guess. All other lines end with a comma, too.

Reply:
He meant the pending comma after `MANAGUA`, but IIRC this is allowed with enums in Java.

Reply:
@xamanu 
> It should be there, i guess. All other lines end with a comma, too.

That's because it's a list of Enum IDs, but all older revisions of this file did not have a pending comma which you just introduced. I think schildbach would like to be consistent and have no pending commas.

Reply:
The current `master` also ends with a comma.

Reply:
I'm fine with the comma at the end of the list. It helps to reduce the diff of future additions.